x-pack/plugins/security_solution/public/common/components/exceptions/exceptionable_fields.json

```diff
@@ -127,5 +127,6 @@
   "user.domain",
   "user.email",
   "user.hash",
-  "user.id"
+  "user.id",
+  "Ransomware.feature"
```

Comment:
Only putting the `Ransomware.feature` field into the exceptionable fields because we use it to prepopulate, in general, we don't want to allow users to allowlist on the other ransomware fields

Reply:
This PR comment might be better left as an inline comment in the code.